### PR TITLE
Geras bug fixes

### PR DESCRIPTION
**Pull request by Just-a-Unity-Dev**
## About the PR

Fixes the following bugs in relation to slimes:
- Zombie origforms can turn into Geras and be unzombified
- Geras do not retain their origform's name
- All AI mobs do not attack Geras
- Geras not having their speed buff

Also nerfs the health of Geras' a bit (120->80)

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
Make sure to take this Changelog template out of the comment block in order for it to show up.
:cl:
- fix: Fixed Geras not having their speed buff.
- fix: Geras are no longer immune to other AI mobs.
- fix: Slimes can no longer morph into a Geras when zombified.
- tweak: Lowered the Geras death threshold.

### CODE REVIEW

**Comment by Djungelskog2** [comment]
adding on, geras cant go through AA doors like carp and diona nymphs can

**Comment by FungiFellow** [comment]
> adding on, geras cant go through AA doors like carp and diona nymphs can

They need `DoorBumpOpener `I think

**Comment by Dutch-VanDerLinde** [comment]
Why do you need to copy the name in the code when you can just enable the transfer name thing with polymorph

**Comment by QuietlyWhisper** [comment]
> Why do you need to copy the name in the code when you can just enable the transfer name thing with polymorph

That's true, it should just work in theory

image

File: Content.Server/Geras/GerasSystem.cs [hunk 1]
```diff
@@ -29,13 +30,22 @@ private void OnMapInit(EntityUid uid, GerasComponent component, MapInitEvent arg
 
     private void OnMorphIntoGeras(EntityUid uid, GerasComponent component, MorphIntoGeras args)
     {
+        if (HasComp<ZombieComponent>(uid))
+            return; // i hate zomber.
+
```
Comment (metalgearsloth) [inline Content.Server/Geras/GerasSystem.cs:35]: Can't you just sub to zombie and remove the action instead of leading a dummy action on them indefinitely.

**Comment by Just-a-Unity-Dev** [comment]
> Why do you need to copy the name in the code when you can just enable the transfer name thing with polymorph

was not aware

**Comment by Just-a-Unity-Dev** [comment]
Reviews are finished, tests are fixed, suggestions are implemented

**Comment by Just-a-Unity-Dev** [comment]
https://github.com/space-wizards/space-station-14/assets/67359748/8fc1c442-72af-4cc1-b22d-4fdf93fa4830

also recorded by the same user, I have *no* clue how any of this can occur

**Comment by Just-a-Unity-Dev** [comment]
waiting for review...

**Review by VasilisThePikachu — APPROVED** [review]
lgtm

**Pull request merged**